README.md

```diff
@@ -0,0 +1,55 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configuration that we use at Feedzai in order check our code conventions and coding style in JavaScript projects. The objective of this package is provide the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
```

Comment:
I don't understand what you mean by: "in order check our code conventions and coding style in JavaScript projects."